core/container.go

```diff
@@ -157,18 +157,43 @@ func (container *Container) FS(ctx context.Context) (*Directory, error) {
 	return NewDirectory(ctx, st, "", payload.Platform)
 }
 
-func (container *Container) WithFS(ctx context.Context, st llb.State, platform specs.Platform) (*Container, error) {
+func (container *Container) FromDirectory(ctx context.Context, id DirectoryID) (*Container, error) {
```

Comment:
I might be missing context but I don't see this is being used right now.

Reply:
You're totally right. I forgot to pull that code out before I pushed.